.rubocop.yml

```diff
@@ -0,0 +1,852 @@
+# all of these could/should be corrected. Have at it if you want.
+
+# Cop supports --auto-correct.
+# Configuration parameters: TreatCommentsAsGroupSeparators, Include.
+# Include: **/*.gemfile, **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
+
+# Cop supports --auto-correct.
+# Configuration parameters: TreatCommentsAsGroupSeparators, Include.
+# Include: **/*.gemspec
+Gemspec/OrderedDependencies:
+  Exclude:
+    - 'omnibus.gemspec'
```

Comment:
we don't ever want to enable these ones...

Reply:
I fixed that whole file up. I think I ran it with rubocop by accident